### ISSUE TITLE
Touch events shouldn't be forwarded to the clients.

### ISSUE THREAD

**Issue by spacepluk** [synergy/synergy#4544]
I'm using a laptop as the synergy server that comes with a touchscreen.  If I touch the screen while the mouse is on one of the clients the "clicks" are forwarded. It's not a big deal but it's not very useful, and sometimes I touch the screen accidentaly while moving the laptop lid which is annoying.

**Comment by speaker** [comment]
This is actually pretty interesting. With the popularity of touch screens we should look into a design that makes sense. On the surface it seems obvious that a touch event wouldn't need to be forwarded.

**Comment by XinyuHou** [comment]
> I would like to make the suggestion to have an option, that allows Synergy to jump to the cursor and change focus to another machine, that is being interacted through a touch screen.
> 
> Example:
> 
> I am surfing the web, finding art references on my larger monitors with my primary workstation. I find something I want to draw. I hover my stylus over my tablet screen and begin to draw. Then I use the same keyboard to attempt to change the brush or settings, but it begins interfering with the browser on my work station instead of the art program. So, I use the mouse to move the cursor all the way over to the tablet and continue to draw.
> 
> This is something I've gotten quite used to, but I imagine would be a very useful and easy option to add.

**Comment by XinyuHou** [comment]
Kinda of an FYI as much as a bug, not sure it can be fixed.

Basically if you have a touch screen device and this is the Synergy Server, if the mouse is moved to another PC then the screen is touched on the Server, the remote PC gets the input rather than the screen you touched! Kinda freaky at first lol. :)

Doesn't matter which synergy version I have just tested it on a few revisions all windows some 32, some 64.

Thanks

Stuart.

>

**Comment by yidhoe** [comment]
Yes a feature which would differentiate touch and mouse wood be perfect. 😀

**Comment by Rophuine** [comment]
This would be really nice. I use Synergy 2 to drive my Surfacebook+Surface Pro setup - I use the Pro for drawing and the surfacebook for lots of other stuff, and Synergy is great because it let's me use the keyboard from my 'book for both devices.

It's really frustrating though that touch and pen input requires me to drag the mouse cursor around. I'd really love to have any touch or pen input point my keyboard at whichever device I'm using, and not forward that event to whichever device currently has focus.

Love your product but this would be a really nice improvement, especially with the increasing prevalence of touch devices.
